vector/v.distance/main.c

```diff
@@ -266,7 +266,8 @@ int main(int argc, char *argv[])
     opt.to_field->guidependency = G_store(opt.to_column->key);
 
     G_option_required(opt.out, opt.table, flag.print, opt.column, NULL);
-    G_option_exclusive(opt.out, opt.table, flag.print, NULL);
+    G_option_exclusive(opt.out, flag.print, NULL);
+    G_option_exclusive(opt.table, flag.print, NULL);
```

Comment:
Wasn't the idea in the original #601 code _G_option_excludes_ (when first is present, none of the other options should be present)?

```c
G_option_excludes(flag.print, opt.out, opt.table, NULL);
```

Reply:
Yes, yu are right. Implemented in 3f7f04f